cmd/testawskms/main.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) MongoDB, Inc. 2022-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func main() {
+	uri := os.Getenv("MONGODB_URI")
+	// expecterror is an expect error substring. Set to empty string to expect no error.
+	expecterror := os.Getenv("EXPECT_ERROR")
+
+	if uri == "" {
+		fmt.Println("ERROR: Please set required MONGODB_URI environment variable.")
+		fmt.Println("The following environment variables are understood:")
+		fmt.Println("- MONGODB_URI as a MongoDB URI. Example: 'mongodb://localhost:27017'")
+		fmt.Println("- EXPECT_ERROR as an optional expected error substring.")
+		os.Exit(1)
```

Comment:
What is the purpose of using `os.Exit` here instead of `return`?

Reply:
Exit(1) indicates `EXIT_FAILURE` following the convention from the UNIX C era though it is not portable across platforms as I remember. I used `os.Exit` here mostly because "cmd/testgcpkms/main.go" does the same way. I'm not sure what an equivalent Golang idiom is.